crates/ruma-state-res/src/lib.rs

```diff
@@ -68,27 +68,37 @@ impl StateResolution {
             return Ok(clean);
         }
 
-        info!("{} conflicting events", conflicting.len());
+        info!("Conflicting events size: {}", conflicting.len());
```

Comment:
This is just a count of events, right? Maybe have the log msg say "Amount of conflicting events" then.